source/utilities.tex

```diff
@@ -18341,10 +18341,27 @@
 if there is a template type argument $\tcode{B}_{i}$
 for which \tcode{bool($\tcode{B}_{i}$::value)} is \tcode{false},
 then instantiating \tcode{conjunction<$\tcode{B}_{1}$, $\dotsc$, $\tcode{B}_{N}$>::value}
-does not require the instantiation of \tcode{$\tcode{B}_{j}$::value} for $j > i$.
+does not require a definition of \tcode{$\tcode{B}_{j}$::value} to exist\iref{temp.inst} for $i < j \leq N$.
 \begin{note} This is analogous to the short-circuiting behavior of
 the built-in operator \tcode{\&\&}.
 \end{note}
+\begin{note}
+This allows types without a \tcode{value} static data member
+to be template arguments for \tcode{conjunction}, as long as an
+earlier argument evaluates to \tcode{false}, for example:
+\begin{codeblock}
+template<typename T>
+  concept HasValue = requires { { T::value } -> bool; };
+template<typename T>
+  using maybe_value = std::conjunction<std::bool_constant<HasValue<T>>, T>;
+
+struct X { };
+
+static_assert( ! maybe_value<X>::value );      // OK, \tcode{X::value} not used
+static_assert( ! maybe_value<int>::value );    // OK, \tcode{int::value} not used
+static_assert( maybe_value<std::true_type>::value );
+\end{codeblock}
+\end{note}
 
 \pnum
 Every template type argument
```

Comment:
GitHub isn't letting me comment on it, but there's a "for which instantiation is required" in here that should also be updated.

Reply:
Sorry, I'm not sure what needs to be updated, or how.

Reply:
Richard can't find those words in the current draft, so let's ignore this comment.